Assets/__Scripts/InputOutputManager/Cube.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace InputOutputManager
+{
+    [Serializable]
+    class Cube
+    {
+        public double x;
```

Comment:
Use Vector3 instead of individual coordinate fields

Reply:
Done